packages/utils/src/TypeUtils.ts

```diff
@@ -1,3 +1,18 @@
+/**
+ * Util type to create a "subtype" of T. Useful for creating subsets of union
+ * types.
+ *
+ * e.g.
+ * declare type Direction = 'north' | 'south' | 'east' | 'west'
+ *
+ * // This works
+ * type Down = Subset<Direction, 'south'> // 'south'
```

Comment:
Assuming these examples should be Extends

Reply:
sigh, yes

Reply:
fixed